README.md

```diff
@@ -1 +1,55 @@
-# LeaderBoard
\ No newline at end of file
+# LeaderBoard
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+### Prerequisites
+
+- Git
+- GitHub
+- Node Js
+- Webpack
+- Windows 10
+
+## Live Demo
+
+[Live Demo Link](Coming soon)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Install
+
+- On GitHub.com, navigate to the main page of the repository. Above the list of files, click Download icon and Code.
+- Copy for git clone
+- Type git clone, and then paste the URL you copied earlier.
+`$ git clone https://github.com/SurafelGetachewT/LeaderBoard.git`
+- Install dependencies `$ npm install`
+- If you want to build the project `$ npm run build` - this will build the project in /dist folder
+- Start the Project `$ npm start` - serve the project in browser
+
+## Author
+
+👤 **Surafel Getachew**
+
+- GitHub: [@SurafelGetachewT](https://github.com/SurafelGetachewT)
+- Twitter: [@SuraGetch](https://twitter.com/SuraGetch)
+- LinkedIn: [LinekdIn](https://www.linkedin.com/in/surafel-getachew-80155b187/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/SurafelGetachewT/LeaderBoard/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](LICENCE.md) licensed.
```

Comment:
- [ ] Please fix your link to your LICENSE file. Replace `[MIT](LICENSE.md)` to `[MIT](./LICENSE)` :blush: